planning/behavior_path_start_planner_module/config/start_planner.param.yaml

```
@@ -38,6 +38,7 @@
       backward_path_update_duration: 3.0
       ignore_distance_from_lane_end: 15.0
       # turns signal
+      prepare_time_before_start: 3.0
```

```suggestion
      prepare_time_before_start: 0.0
```
Same as proposal from @.kosuke55 0.0 by default is natural for me.